app/components/work_packages/activities_tab/journals/new_component.sass

```diff
@@ -1,11 +1,17 @@
+@import "helpers"
+
 .work-packages-activities-tab-journals-new-component
     &--input-trigger-column
         width: 100%
+        overflow: hidden
         button
+            min-width: 0
             background: var(--bgColor-default)
             cursor: text
-            .Button-content
+            span
```

Comment:
We need to make all child spans, not just the `.Button-content`, behave like blocks otherwise they never respect the space available